af_pipeline/af_input/template.py

```
@@ -0,0 +1,281 @@
+    ## Returns:
+
+    - **tuple**:<br />
+        A tuple containing two lists:
+        - List of query indices corresponding to the aligned template residues.
+        - List of template indices corresponding to the aligned query residues.
+    """
+
+    # Perform pairwise alignment
+    aligner = PairwiseAligner(scoring="blastp")
+    alignments = aligner.align(seqA=query_seq, seqB=template_seq)
+    alignment = list(alignments)[0]  # Take the best alignment
```

Ruff (0.16.3)

[warning] Prefer `next(iter(alignments))` over single element slice

Replace with `next(iter(alignments))`

(RUF015)